src/cases/converters/case-converter.ts

```diff
@@ -12,3 +12,15 @@ export function toShareCaseConverter(selectedCases: SearchResultViewItem[]): Sha
   }
   return sharedCases;
 }
+
+export function toSearchResultViewItemConverter(shareCases: SharedCase[]): SearchResultViewItem[] {
+  const searchResultViewItems: SearchResultViewItem[] = [];
+  for (const shareCase of shareCases) {
+    const searchResultViewItem = {
```

Comment:
You could do this as you are not using the reference later.
searchResultViewItems.push({
      case_id: shareCase.caseId,
      case_fields: null
    });